cypress/e2e/google.cy.js

```diff
@@ -0,0 +1,7 @@
+describe("This is my first cypress test", () => {
+    it("should have a title", () => {
+      cy.visit("https://www.google.com/");
+      cy.title().should("eq", "Google");
+    });
+  });
+  
```

Comment:
Please fix EOF error

GitHub will show you that your file is missing a newline at the end of the file with a little red marker (that if you hover over it, it will say "No newline at end of file"). You should see this marker and correct it before asking for a review.

Reply:
Fixed, thanks a lot